ts/node/sql.ts

```diff
@@ -458,10 +458,6 @@ function saveConversation(data: ConversationAttributes, instance?: BetterSqlite3
     conversationIdOrigin,
   } = formatted;
 
-  // shorten the last message as we never need more than 60 chars (and it bloats the redux/ipc calls uselessly
```

Comment:
Are we just ignoring this issue now?

Also I thought it was an alignment problem.